File: 020-reverse-sentence/reverseSentence.js
function reverseSentence(str) {
  const newStr = str.split(" ").reverse().join(" ");
  return newStr
}

console.log(reverseSentence("world hello"))
console.log(reverseSentence("push commit git"))
console.log(reverseSentence("npm install sudo"))
console.log(reverseSentence("import default function export"))

File: 020-reverse-sentence/reverse_sentence.py
def reverse_sentence(input_str):
  str_list  = input_str.split(" ")
  str_list.reverse()
  new_str = " ".join(str_list)
  return new_str

print(reverse_sentence("world hello"))
print(reverse_sentence("push commit git"))
print(reverse_sentence("npm install sudo"))
print(reverse_sentence("import default function export"))